platform-ui-skin/src/main/webapp/skin/less/social/skin/popover/popover.less

```
@@ -54,3 +55,8 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     max-width: inherit !important;
   }
 }
+
+.profile-popover-menu { 
+	height: 160px!important;
+	box-shadow: none!important;
```

Please, avoid using tabs for indentation
```suggestion
    height: 160px!important;
    box-shadow: none!important;
```

---

done!!